src/test/java/uk/gov/justice/digital/hmpps/vsip/services/PrisonVisitsTestingHelperService.java

```diff
@@ -114,6 +115,22 @@ public void changeVisitPrison(String bookingReference, String prisonCode) {
         LOG.debug("Exit changeVisitPrison");
     }
 
+    public void createSessionTemplate(String prisonCode,
+                                      LocalDateTime sessionStartDateTime,
+                                      Integer weeklyFrequency,
+                                      Integer closedCapacity,
+                                      Integer openCapacity,
+                                      String locationLevels,
+                                      String incentive,
+                                      String category,
+                                      Boolean disableAllOtherSessionsForSlotAndPrison) {
+        LOG.debug("Enter changeVisitPrison {1} {2} {3} {4} {5} {6} {7} {8} {9}", prisonCode, sessionStartDateTime,weeklyFrequency,closedCapacity, openCapacity,locationLevels,incentive,category,disableAllOtherSessionsForSlotAndPrison);
```

Comment:
The debug format is {} without the numbers.

Reply:
updated